helm/cron_jobs/replayer-cronjobs/templates/gcloud-secret.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: gcloud-keyfile
```

Comment:
If there is an existing `gcloud-keyfile` secret in the destination namespace then applying this manifest will cause an error.

I would suggest simply to use `.metadata.name: gcloud-keyfile-cronjobs` to avoid these conflicts.